test/ex_plasma/transaction_test.exs

```diff
@@ -4,6 +4,53 @@ defmodule ExPlasma.TransactionTest do
 
   alias ExPlasma.Transaction
 
+  describe "new/1" do
+    test "does not allow amount to be zero" do
+      rlp = [
```

Comment:
same tests but with Utxos structs?

Reply:
Yeah, these are the same tests but with `Transaction` structs. Want to confirm we propogated these errors upwards. Any recommendations here?

Reply:
not sure I understand. Recommendation for what?